docs/release_docs/AHA-Prod_ReleaseFlow.md

```diff
@@ -0,0 +1,36 @@
+# AHA-Prod Workflow
+Explaination of AHA-PROD workflow.
+![AHA-PROD](https://github.com/REAN-Foundation/reancare-service/blob/feature/flow_documentation/assets/images/AHA-PROD_Workflow.png?raw=true)
+
+## AHA-PROD-ci-cd
+
+This workflow allows you to manually trigger a GitHub Action with a input Github ``` Tag_name ```, without having to push or create a pull request
+
+### JOBS
```

Comment:
Provide some brief info like: "this workflow uses two jobs: GitHub-ECR-Tag-Check and Deploy ECS to verify and deploy `reancare-service` release to `aha-prod` environment

Reply:
This comment is still pending, did we make any updates?